libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -5,11 +5,14 @@
       matInput
       type="search"
       placeholder="Search for books to add to your reading list"
-      formControlName="term"
-    />
-    <button mat-icon-button matSuffix>
+      aria-label="Search for books to add to your reading list"
```

Comment:
This sounds more like a button label than the label for an input field.

Reply:
Made the appropriate change